src/components/ItemListContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+    ListHeader,
+    ListContainer,
+    DeleteBtn,
+    List
+}from './ItemListPresenter';
+
+
+const TodoList = (props) => {
```

Comment:
```suggestion
const TodoList = ({onToggle, onDeleteBtnPressed}) => {
```

이런식으로 어떤 `props`가 부모 컴포넌트로 넘어왔는지 명시해주면 더 좋겠죠!